r/R/dplyr-functions.R

```diff
@@ -250,6 +250,39 @@ nse_funcs$is_logical <- function(x, n = NULL) {
   nse_funcs$is.logical(x)
 }
 
+# Create a data frame/tibble/struct column
+nse_funcs$tibble <- function(..., .rows = NULL, .name_repair = NULL) {
+  if (!is.null(.rows)) arrow_not_supported(".rows")
+  if (!is.null(.name_repair)) arrow_not_supported(".name_repair")
+
+  # use dots_list() because this is what tibble() uses to allow the
+  # useful shorthand of tibble(col1, col2) -> tibble(col1 = col1, col2 = col2_)
```

Comment:
```suggestion
  # useful shorthand of tibble(col1, col2) -> tibble(col1 = col1, col2 = col2)
```

Is this `_` a typo? I think yes, but not 100% certain it's meaningful